### ISSUE TITLE
Can't install into arduino IDE 1.6.12

### ISSUE THREAD

**Issue by ebeyer** [PaulStoffregen/Time#54]
The installer refuses to install libraries into the latest arduino IDE (v1.6.12, as of this post).  It supports v1.6.11.

**Comment by PaulStoffregen** [comment]
![sc](https://cloud.githubusercontent.com/assets/965463/18805180/ac8c9eee-81bb-11e6-9a11-681299a456fb.png)

**Comment by ebeyer** [comment]
But how does that help me _now_?   :)

Seriously, though.  I'm looking forward to using your library for this really cool project (https://learn.adafruit.com/flora-geo-watch/overview), so thanks for all of your work.
EB

**Comment by PaulStoffregen** [comment]
Well, you click that link, then on the page with the beta versions, you download the one you need (depending on your operating system).  Then run the installer you just downloaded.  It will recognize and install into Arduino 1.6.12.  Then run 1.6.12, select Teensy from the Tools - Boards menu, and use it normally.

How can I improve the website to make this clearer?

**Comment by ebeyer** [comment]
I'm really sorry.  Your web site is plenty clear.  But I got to the downloader by way of the Adafruit project I linked you, which takes you to its github page, which directs you to https://www.pjrc.com/teensy/td_libs_Time.html.  If you forge ahead to the download link, as I did, and not to the Github one, you get the non-beta version.  I'll give the beta a go and let you know if there are issues.

Thanks again.
